plugins/modules/atlas_user.py

```diff
@@ -76,6 +76,25 @@
     required: true
     type: list
     elements: dict
+  scopes:
+    description:
+      - List of clusters and Atlas Data Lakes that this user can access.
+      - Atlas grants database users access to all resources by default.
+    suboptions:
+      name:
+        required: true
+        type: str
+        description:
+          - Name of cluster or Atlas Data Lake.
+      type:
+        type: str
+        choices: ["CLUSTER", "DATA_LAKE"]
+        default: "CLUSTER"
+        description:
+          - Either "CLUSTER" or "DATA_LAKE" depending on target.
```

Comment:
The description is redundant with the choices.. Is there maybe a better description of `type`?

Reply:
better?

Reply:
better!